registry.json

```diff
@@ -1,5 +1,19 @@
 [
    {
+    "ticker": "FERAL",                                                       
+    "poolId": "c8b9f43186ff7a30a400a37e81b677f71e88eaee90250cf9462abfc9", 
+    "social": {       
+      "website": "https://feral-ada.co.uk",
+      "twitter": "FeralPool", 
+      "instagram": "",
+      "youtube": "",
+      "facebook": ""
+    },
+    "telegram": "Feral_pool",                                          
```

Comment:
I cant find this username on telegram. You must include a telegram username and it must be the username of an individual and not a group or channel